messages.md

```diff
@@ -170,6 +170,9 @@ As each wallet will verify and store signatures locally, the server isn't truste
 managed by the organisation deploying Revault itself or any third party without risking any
 loss of funds.
 
+The server will also broadcast the spend transactions when the unvault timelock is expired
+in a deployment with cosigning servers.
```

Comment:
We'd do so in any deployment? Also may be worth adding a sentence with the rationale for doing so

Reply:
> We'd do so in any deployment?

Ah, right, I updated the docs before our chat :)